### PR TITLE
Sam's bio and pic

### PR DESCRIPTION
**Pull request by srlt42**

### CODE REVIEW

File: content/who-we-are/our-team/contents.lr [hunk 1]
```diff
@@ -157,6 +157,29 @@ caption:
 background_color: 
 -----
 padding: 
+#### subnav-item ####
+body:
+
+Hey there, my name is Sam Larimore
+
+I was born in a small town in Missouri and was the youngest of 5, my oldest brother being 10 years older than me. As a result I was very young when he started getting into Engineering and that is what began my interests in all things science and mathematics. As a kid I was a member of every educational extracurricular I could get my hands on from science olympiad to math relays to scholar bowl and more. Eventually in high school this wasn’t enough and I started teaching myself things like trigonometry and physics in my free time.
```
Comment (BenjaminEchelmeier) [inline content/who-we-are/our-team/contents.lr:165]: Science Olympiad, Math Relays, Scholar Bowl

File: content/who-we-are/our-team/contents.lr [hunk 2]
```diff
@@ -157,6 +157,29 @@ caption:
...
+
+Existing in these kinds of circles led me to many of my other interests that still persist to this day. Things like theatre which I did for several years, or Dungeons and Dragons which to this day I am still writing up new stories for every day.
```
Comment (BenjaminEchelmeier) [inline content/who-we-are/our-team/contents.lr:167]: Could grow this a bit to include current hobbies?

File: content/who-we-are/our-team/contents.lr [hunk 3]
```diff
@@ -157,6 +157,29 @@ caption:
...
+
+Eventually I went on to study at Missouri University of Science and Technology where my love of video games and internet culture led me to study computer science. There I made many of my closest friends and began my foundations in understanding and loving the unique problem solving inherent to that world. To this day that still remains my favorite aspect of working with computers.
```
Comment (BenjaminEchelmeier) [inline content/who-we-are/our-team/contents.lr:169]: This paragraph is solid

**Review by BenjaminEchelmeier — CHANGES_REQUESTED** [review]
I think I'd like to see a bit of a reformatting--clarify your paragraph divisions a bit? 
Something like:

1. Where I came from
2. What I do for fun
3. How I came to/why I do this work

**Comment by alan-cugler** [comment]
I think its good!

**Pull request merged**